src/status_im2/common/home/banner/view.cljs

```diff
@@ -78,4 +81,7 @@
 
 (defn set-scroll-shared-value
   [{:keys [shared-value scroll-input]}]
-  (reanimated/set-shared-value shared-value scroll-input))
+  (reanimated/set-shared-value shared-value scroll-input)
+  (let [new-overflow (if (<= scroll-input 3) :visible :hidden)]
```

Comment:
what is `3` referring to? perhaps you could put it in a named def to provide some context of what this value represents

Reply:
@J-Son89 it's a threshold, sure will move it to a def